code/Backend/flask-backend/app.py

```diff
@@ -256,7 +256,7 @@ def login():
         conn = psycopg2.connect(**db_connection_settings)
         cursor = conn.cursor()
         form = LoginForm(meta={'csrf': False})
-
+        session['username'] = None
```

Comment:
Is 'session' already initialized by Flask?

Reply:
No i don't think so - I had to add it to the import at the top